content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. Our code intelligence platform includes a dozen new features to grok code, spend more time in flow, and execute big decisive refactors across your codebase.
+
+A code intelligence platform indexes and understands all of your organization's code, no matter where it's stored or what language it's in. With Sourcegraph, devs and engineering leaders have a complete understanding of their codebase, so they can learn about code and operate more efficiently. 
+
+Read on to discover everything that shipped in Sourcegraph 4.0.  
+
+### 🧠 Code intelligence: uplevel your code search
+
+<br />
+<a href="/code-insights"><Badge text="Code Insights" color="green" size="small" /></a>
+
+#### Understand usage and search structure with Code Insights dashboards within the search UI
+[Code Insights](https://about.sourcegraph.com/code-insights) turns your code into a queryable database to create customizable, visual dashboards. We're bringing the power of Code Insights directly into the search UI to surface high-level information about your code, help you answer questions about your code, and refine your search results.  
+
+Search aggregation charts appear on the search results page to help you answer questions, like:
+- How many different versions of a library or package are present in your code, and which is most common? 
+- Which files have the majority of these search results?
+- Which repositories are this library used in most?
+- Which library components are used most? By who?
+- What are all the possible arguments we pass to this function? 
+
+You can choose to group your search results by location (repo or file), author, or arbitrary capture group pattern. Here are some [example search aggregations to get started with.](https://docs.sourcegraph.com/code_insights/references/search_aggregations_use_cases)
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Search_Aggregations_4.0',
+    mp4: 'blog/release-post/4.0/Search_Aggregations_4.0'
+  }}
+  loop={true}
+  title="High-level aggregations"
+  caption="The power of Code Insights is now within the search UI to surface usage patterns about your code"
+/>
+<a href="https://docs.sourcegraph.com/code_insights/explanations/search_results_aggregations" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+  
+<br />
+<a href="/code-search"><Badge text="Code Search" color="cerise" size="small" /></a>
+
+#### A faster, simpler search experience 
```

Comment:
@limitedmage Could you please review this section by EOD - keep in mind this is for accuracy and that we're focusing on the value to our customers.

Reply:
This has been reviewed by Juliana and the edits have been incorporated.